eth/api.go

```diff
@@ -543,3 +543,42 @@ func (api *PrivateDebugAPI) getModifiedAccounts(startBlock, endBlock *types.Bloc
 	}
 	return dirty, nil
 }
+
+type AccessListResult struct {
```

Comment:
Pls make this private, we don't want to litter the Go API namespace.

Reply:
Done, sorry I thought they needed to be exported for the js to work